README.md

```diff
@@ -157,18 +157,60 @@ the file manually.
 
 ### Spacemacs
 
-Most probably you already have a private `org-roam` layer, if not, see examples
+From version 0.300.0, Spacemacs offers support for `org-roam`. This means private layers are no longer necessary.
+If you have already configured `org-ref`, you can follow this [blog post](https://philipperambert.com/Installing-Org-Roam-Bibtex-In-Spacemacs).
```

Comment:
I think the blog post should be explicitly credited as the source of information.